data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -193,7 +198,12 @@ then
 			--machine-config-operator-image=${MACHINE_CONFIG_OPERATOR_IMAGE} \
 			--machine-config-oscontent-image=${MACHINE_CONFIG_OSCONTENT} \
 			--infra-image=${MACHINE_CONFIG_INFRA_IMAGE} \
-			--cloud-config-file=/assets/manifests/cloud-provider-config.yaml
+			--cloud-config-file=/assets/manifests/cloud-provider-config.yaml \
+			--keepalived-image=${MACHINE_CONFIG_KEEPALIVED_IMAGE} \
+			--coredns-image=${MACHINE_CONFIG_COREDNS_IMAGE} \
+			--mdns-publisher-image=${MACHINE_CONFIG_MDNS_PUBLISHER_IMAGE} \
+			--haproxy-image=${MACHINE_CONFIG_HAPROXY_IMAGE} \
+			--baremetal-runtimecfg=${MACHINE_CONFIG_BAREMETAL_RUNTIMECFG_IMAGE}
```

Comment:
why is there not `image` prefix here in the flag?

Reply:
I copied the flag structure like the `--machine-config-$x-image` above.

Reply:
oh I see you mean the '-image' suffix specifically on baremetal-runtimecfg. I fixed it here and in the mco patch. Thanks!